src/App.vue

```diff
@@ -112,16 +112,16 @@ export default class App extends Vue {
     this.$router.push({ name: "root" });
   }
 
-  updateRightPanel(panel: string) {
+  toggleRightPanel(panel: string) {
     this.$data[panel] = !this.$data[panel];
     // The last panel updated has to be closed if it is not the currently updated panel
     if (
-      this.lastRightPanelUpdate !== undefined &&
-      this.lastRightPanelUpdate !== panel
+      this.lastModifiedRightPanel !== null &&
```

Comment:
`this.lastModifiedRightPanel != null` to compare with null AND undefined

Reply:
It is often considered better practice to be explicit about comparing null and undefined (e.g., `foo !== null && foo !== undefined`.  The 2021 spec for ES adds the new nullish operator (`??`) to confuse things further.

Reply:
The declaration of `lastModifiedRightPanel` is the following: `lastModifiedRightPanel: string | null = null;`
It means that it shouln't be possible for `lastModifiedRightPanel` to be `null`
Should I edit anyway to compare against both `null` and `undefined`?

Reply:
> It means that it shouln't be possible for lastModifiedRightPanel to be `null`

You meant "to be `undefined`", right ?

> It is often considered better practice to be explicit about comparing null and undefined

I personally find it too verbose, error prone (if developer forgets that the variable can become undefined) and potentially harmful when it comes to performance.

Now, in a typescript environment things are a bit different. I lack experience here.

I guess it is all a matter of making a choice (setup linter rules) and enforcing it all over the project. I can see both `!=`/`==` vs `!==`/`===` being used in the code. Your call.

And back to this very specific issue, because you enforced the type to be `string | null`, I guess that gives you enough safety to write `!== null`, sorry about this fuzz.

Reply:
> You meant "to be `undefined`", right ?

Yes, sorry for the mistake!

I'll keep it as it is right now then, thank you!